templates/Kibana.logrotate.j2

```diff
@@ -0,0 +1,22 @@
+{{ kibana_log_dir }}/kibana.stdout
+{
+    rotate 6
+    daily
+    size 100M
```

Comment:
can we change this to `maxsize`, it means it will always rotate when it hits the limit. `size` doesn't do that.